frontend/src/components/common/BoolCheckbox.vue

```diff
@@ -0,0 +1,51 @@
+
+<!-- This is a boolean checkbox (true/false) -->
+
+<script setup lang="ts">
+import Checkbox from 'primevue/checkbox';
+
+// value to bind parent model.
+const model = defineModel()
+
+// configurable options and defaults for props
+interface CheckboxProps {
+    id?: string;
+    label?: string;
+    lfontSize?: string;
+    lfontWeight?: string;
+}
+const props = withDefaults(defineProps<CheckboxProps>(), {
+    id: "checkbox",
+    lfontSize: "0.875rem",
+    lfontWeight: "600"
```

Comment:
Hi Ian, do we need these to be props? I feel we can add them when we need them, for now maybe then can just be local variables

Reply:
I see checkbox might soon be used in other places so adding them for convenience just for fontsize and fontweight. I can remove them, but I will keep the id (with default) if that's ok?

Reply:
Font ones are removed but kept the id.